sample-apps/generic_deepgrow/README.md

```diff
@@ -2,9 +2,73 @@
+  > - [LoadImaged](https://docs.monai.io/en/latest/_modules/monai/transforms/io/array.html#LoadImage) -> Loads both image and label from a dictionary
+  > - [AsChannelFirstd](https://docs.monai.io/en/latest/transforms.html#aschannelfirstd) -> Transforms both image and label to have the channel as first dimension
+  > - [Spacingd](https://docs.monai.io/en/latest/_modules/monai/transforms/spatial/dictionary.html#Spacingd) -> Resample image and label to a different image space,
+  > - [Orientationd](https://docs.monai.io/en/latest/_modules/monai/transforms/spatial/dictionary.html#Orientationd) -> Reorient image and labels,
+  > - [AddChanneld](https://docs.monai.io/en/latest/_modules/monai/transforms/utility/array.html#AddChannel) -> Add an extra channel to the image and label,
+  > - [SpatialCropForegroundd](https://docs.monai.io/en/latest/transforms.html#cropforegroundd) -> This transform randomly changes image intensity by a gamma value, 
+  > - [Resized](https://docs.monai.io/en/latest/_modules/monai/transforms/spatial/dictionary.html#Resized) -> Resample image to a different size,
+  > - [NormalizeIntensityd](https://docs.monai.io/en/latest/_modules/monai/transforms/intensity/dictionary.html#NormalizeIntensityd) -> Normalize intensity of the image,
+
+
+  Transformations used for the clicks simulation [(DeepGrow App)](https://docs.monai.io/en/latest/apps.html)
+  > 
+  > - [FindAllValidSlicesd](https://docs.monai.io/en/latest/_modules/monai/apps/deepgrow/transforms.html#FindAllValidSlicesd) -> This transform finds/lists all valid slices in the label. Label is assumed to be a 4D Volume with shape CDHW, where C=1,
+  > - [AddInitialSeedPointd](https://docs.monai.io/en/latest/_modules/monai/apps/deepgrow/transforms.html#AddInitialSeedPointd) -> This transform adds random guidance as initial seed point for a given label,
+  > - [AddGuidanceSignald](https://docs.monai.io/en/latest/_modules/monai/apps/deepgrow/transforms.html#AddGuidanceSignald) -> This transform adds guidance signal (foreground and background clicks) for input image, 
+  > - DiscardAddGuidanced -> This transform discards foreground and background tensors with a probability define as an argument. 
+      > This transform is responsible for the training of both automated model and interactive model. 
+      > This means, sometimes the input is a tensor with only the image and two empty tensors and sometimes is a tensor with the image and the foreground and backgounr points. 
+
+- Number of epochs
+  > Default value is 50 Epochs for DeepGrow 3D and 20 Epochs for DeepGrow 2D. Change this in the [info.YAML](./info.yaml) file.
+
+- learning rate
+  > Default value is 0.0001. Change this in the [info.YAML](./info.yaml) file
+
+- validation split used during training
+    > Default value is 0.2. Change this in the [info.YAML](./info.yaml) file
+
+### Inputs
+
+- 1 channel for the image modality -> Automated mode
```

Comment:
Does deepgrow have this case? I think it always has three channels as input only, right?

Reply:
The user only has to deal with a single channel.

Reply:
By channel, I didn't mean modalities. That's why I thought deepgrow only works with three channels/tensors (image, foreground points and background points) not one. Isn't it?

Reply:
Removed